internal/cli/opsmanager/versionmanifest/update.go

```diff
@@ -41,7 +41,7 @@ func (opts *UpdateOpts) initStore() error {
 	if err != nil {
 		return err
 	}
-	opts.storeStaticPath, err = store.NewStaticPath(config.Default())
+	opts.storeStaticPath, err = store.NewForVersionManifest(config.Default())
```

Comment:
[nit - feel free to ignore] Could we rename `NewForVersionManifest` method something like `NewVersionManifestPath`? `NewForVersionManifest ` method name is confusing to me.

Reply:
yep, I don't like it either, I've been wanting to do a refactor on how we set up the store but I'll leave that for another time